src/tsconfig.ts

```diff
@@ -147,13 +147,13 @@ export function parseFileSync (contents: string, filename: string): TSConfig {
  * Sanitize a configuration object.
  */
 export function resolveConfig (data: TSConfig, filename: string, cb: (err: Error, config?: TSConfig) => any) {
-  const filesGlob = getGlob(data)
+  const [filesGlob, opts] = getGlob(data)
 
   if (filesGlob) {
-    return glob(filesGlob, {
+    return glob(filesGlob, extend(opts, {
```

Comment:
Isn't `filesGlob` going to be an array? From the `glob` docs it needs to be a `string` : https://github.com/isaacs/node-glob#globpattern-options-cb